lib/jquery.ui/jquery.ui.languagesuggester.js

```diff
@@ -1,15 +1,42 @@
 ( function( $ ) {
 	'use strict';
 
+var PARENT = $.ui.suggester;
```

Comment:
This could be it's own patch (along with your name change and maybe a few other lines in this patch), but is probably not worth it.